source/DoorNode.cpp

```diff
@@ -0,0 +1,42 @@
+﻿#include "DoorNode.h"
+
+#include <cugl/2d/CUAnimationNode.h>
+
+using namespace cugl;
+
+/** The diameter of the ship. Also the x coordinate of the center of the ship */
+constexpr unsigned int DIAMETER = 1280;
```

Comment:
At some point we should probably consider factoring out all these constants into a centralized location. This can stay for now, though.